server/models/userModels.js

```diff
@@ -1,14 +1,36 @@
 import mongoose from 'mongoose';
 import { v4 as uuidv4 } from 'uuid';
+import validator from 'validator';
```

Comment:
validator is imported here and used on userModels.js, but it seems like validator is not installed? (I didn't see it on package.json so I installed it to test your code)

Reply:
It seems it's working without installing the package...wierd!
I installed the package anyway.

Thanks for checking 🙏